tests/test_elasticsearch.py

```diff
@@ -43,7 +45,7 @@ def setUp(self):
     def test_info(self):
         self.es._async_http_client.request = MagicMock(return_value=self.generate_response(ResponseCodes.OK))
         yield self.es.info()
-        expected_url = '{host}:{port}/'.format(host=SOME_HOST, port=SOME_PORT)
+        expected_url = '{host}:{port}/'.format(host=SOME_HOST, port=SOME_PORT).encode(UTF_8_ENCODING)
```

Comment:
method to create the expected url that encodes with UTF_8 and creates the url based on the params